src/logger.zig

```diff
@@ -700,7 +707,7 @@ pub const Log = struct {
         });
     }
 
-    pub fn toJS(this: Log, global: *JSC.JSGlobalObject, allocator: std.mem.Allocator, fmt: string) JSC.JSValue {
+    pub fn toJSError(this: Log, global: *JSC.JSGlobalObject, allocator: std.mem.Allocator, fmt: string) JSC.JSValue {
```

Comment:
```suggestion
    pub fn toJS(this: Log, global: *JSC.JSGlobalObject, allocator: std.mem.Allocator, fmt: string) JSC.JSValue {
```

the only usage where we don't want to throw it is in the bundler

let's keep it as `toJS` for consistentcy with the rest of the code